code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -443,9 +444,9 @@
 		)
 
 /obj/item/storage/belt/janitor/full/populate_contents()
-	new /obj/item/lightreplacer(src)
```

Comment:
Why take away the closet's light replacer? If this is your intention, the removal needs to be listed in the changelog.

Reply:
All janitorial ERTs spawns with this belt, to avoid creating different versions i swapped the light replacer with an advanced cleaner from the backpack, leaving Red/Amber with common replacer and gamma with the bluespace one.

Reply:
Ok, I see now, nevermind.